package.json

```diff
@@ -1,4 +1,7 @@
 {
+	"dependencies": {
+		"node-fetch": "^2.6.0"
```

Comment:
Just to say, Origami components can not have dependencies which come from npm as the component has to work via Bower.

You can remove this dependency completely as Origami components are running in a Web Browser and browsers already have `fetch` available.

Reply:
Good to know 👍 I'll remove it